cmd/event-logger/main.go

```diff
@@ -0,0 +1,166 @@
+// Event logger is a tool to capture sqs events and display the most recent as a
+// HTML page. It keeps the last 10 messages received, and those results can be
+// filtered using ?detail-type and ?detail query parameters. It will wait 10
+// seconds for a result when filtering.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/aws-sdk-go-v2/service/sqs/types"
+	"github.com/ministryofjustice/opg-go-common/env"
+)
+
+type message struct {
+	Time       time.Time
+	Detail     json.RawMessage
+	DetailType string `json:"detail-type"`
+	Source     string
+}
+
+const messageCount = 10
```

Comment:
I had to parse around the code below a bit to work out the purpose of this - would `messageLimit` or `maxMessages` be any clearer on intent?